examples/analytics.amp.html

```diff
@@ -442,6 +442,18 @@
 </script>
 </amp-analytics>
 <!-- End SOASTA mPulse example -->
+
+<!-- Linkpulse example - please replace id with your Linkpulse id -->
+<amp-analytics type="linkpulse">
+  <script type="application/json">
+     {
```

Comment:
No worries. My error, I should have checked the example when I corrected the formatting.